lib/run-status.js

```diff
@@ -139,6 +140,7 @@ class RunStatus extends EventEmitter {
 		this.skipCount = sum(this.stats, 'skipCount');
 		this.todoCount = sum(this.stats, 'todoCount');
 		this.failCount = sum(this.stats, 'failCount');
+		this.remainingCount = (this.testCount - (this.passCount + this.failCount));
```

Comment:
`this.testCount - this.passCount - this.failCount` is clearer.

You need to include other test states in this count, see the highlighted lines in https://github.com/avajs/ava/blob/033d4dcdcbdadbf665c740ff450c2a775a8373dc/lib/run-status.js#L41:L45.

The computation should be tested. That's probably best in `test/run-status.js`, though most of its behavior ends up being tested elsewhere.